relational_operators/WorkOrder.proto

```diff
@@ -44,6 +44,7 @@ enum WorkOrderType {
   UPDATE = 20;
   WINDOW_AGGREGATION = 21;
   DESTROY_AGGREGATION_STATE = 22;
+  BUILD_AGGREGATION_EXISTENCE_MAP = 23;
```

Comment:
Please sort the type in the alphabet order (similarly to `BuildAggregationExistenceMapWorkOrder` below), and add the following comment above `enum WorkOrderType {`:

```
// Next tag: 24.
enum WorkOrderType {
```

Recall that this is how Google deals with changing proto definitions. And `Next tag` is the next available number to assign to a new type. After adding the new type, we need to increase `Next tag` by one.